manifests/init.pp

```diff
@@ -22,7 +22,10 @@
   String $key_file = "/etc/puppetlabs/puppet/ssl/private_keys/${trusted['certname']}.pem",
   String $ca_file = "/etc/puppetlabs/puppet/ssl/certs/ca.pem",
   Boolean $manage_collectd = false,
-  Optional[Integer] $limit_nofile = undef
+  Optional[Integer] $limit_nofile = undef,
```

Comment:
rather than the extra flag could we detect when the `$user` and `$group` is non root and infer this?

I am conscious of flag scrawl :)